src/main/java/uk/gov/hmcts/reform/sscs/ccd/presubmit/actionfurtherevidence/HandleEvidenceEventHandler.java

```diff
@@ -125,6 +132,18 @@ private SscsDocument buildSscsDocument(SscsCaseData sscsCaseData, ScannedDocumen
 
         DocumentLink url = scannedDocument.getValue().getUrl();
 
+        log.info("document link: " + url);
+
+        if (caseState != null && isIssueFurtherEvidenceToAllParties(sscsCaseData.getFurtherEvidenceAction())
+                && (caseState.equals(State.DORMANT_APPEAL_STATE)
+                || caseState.equals(State.HEARING)
```

Comment:
Hearing isn't a valid state. You might want to run this past Gareth but it doesn't make any sense to have this as a state anyway. Suggest it is removed from the AC in the ticket but double check